src/jobs/candlepinInternationalizationJob.groovy

```diff
@@ -0,0 +1,17 @@
+import jobLib.rhsmLib
+
+String baseFolder = rhsmLib.candlepinJobFolder
+
+job("$baseFolder/candlepin-internationalization") {
+    description('Periodically executes the gettext and msgattrib tasks on master/hotfix branches')
+    label('rhsm')
```

Comment:
One issue we need to solve: This job will need to run on a node that has both java 11 and 8, and the nodes denoted by this label don't have java 11 afaik.

@Lorquas Would it be better if we use the `candlepinbuild` label (I think this one is used in the release job that has both versions installed)?

Reply:
that should work for now

Reply:
Updated.